packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -167,6 +173,63 @@ const sharedStyles = StyleSheet.create({
     },
 });
 
+const action = semanticColor.action.outlined.progressive;
+
+// NOTE: This color is only used here.
+const pink = "#fa50ae";
+
+/**
+ * TODO(WB-1862): Move this to a shared theme file.
+ */
+const theme = {
+    color: {
+        // Primary link color
+        default: {
+            rest: {
+                foreground: action.default.foreground,
+            },
+            hover: {
+                foreground: action.hover.foreground,
+            },
+            focus: {
+                border: semanticColor.border.focus,
+                foreground: action.hover.foreground,
```

Comment:
Maybe this is an example where `action.focus.foreground` would be useful! If we change hover styles, would we expect focus styles to change too?